### DISCUSSION TITLE
cn/2024/11/28/ci/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#190] [Announcements]
# cn/2024/11/28/ci/

https://hongtaoh.com/cn/2024/11/28/ci/

**Comment by wangyunzi** [comment]
你的标题让我很有点进来阅读的冲动。

**Reply by hongtaoh** [reply]
是有点标题党了。。

**Comment by Liam-Zhong** [comment]
>互联网贵在公开，而许多中文原创内容却无>法通过浏览器搜索到，这实在是一个讽刺

博主所说的应该是小红书公众号的内容？如果是的话这点全球都很公平，推特油管不登录一样看不到，而且圈地算是趋势。偏抽象的学术或者技术上的东西国内的确有待提升。但我认为这不单是互联网的问题，毕竟个人网站在百度收录还得实名……

另外，我比较少看搜狐和网易，偶尔搜搜人文相关的东西还是有很多基础的入门介绍。原创我倒不是特清楚，相比之下 quora 和 YouTube 莫名其妙的话题推送我觉得也挺没有营养的。

**Reply by hongtaoh** [reply]
>推特油管不登录一样看不到

这一点我实在没办法同意。推特是可以搜到的，比如谷歌搜 elon musk，第二条就是他的 x 账号。youtube 更是完全不用登陆就可以看到全部内容。

**Reply by Liam-Zhong** [reply]
啊？可能你人在国外吧…我很多同学都反应 YouTube 都要登陆。你说的搜得到确实如此，但在我这里都强制登陆。

**Reply by hongtaoh** [reply]
有可能。

**Comment by weifanz** [comment]
另一个悲哀是墙内论坛的匿名模式消失了

**Reply by hongtaoh** [reply]
这个没办法。
